### DISCUSSION TITLE
Disable swipe for modes switch

### DISCUSSION THREAD

**Discussion by ruslanhirychau** [gopro/labs#374] [Feature Requests]
Hi there,
Is it possible to disable horizontal swipes for changing modes?
Several times I accidentally have recorded Timelapse instead of Video.
I would be happy to have mode switching only with the Mode key.

**Comment by dnewman-gpsw** [comment]
This is a common, and good, request. Currently this is no simple solution to have video only in the carousel.  You can lock the display, that way only mode key will switch modes.

**Comment by tedder** [comment]
Per dnewman's suggestion](https://github.com/gopro/labs/discussions/346) I'm experimenting with archive mode: https://gopro.github.io/labs/control/archive/
